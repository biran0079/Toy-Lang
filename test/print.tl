print(1);
print(2);
print(3);
print(print(3));
print(1,3,5,7);
