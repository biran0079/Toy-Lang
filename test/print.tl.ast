[stmts, [call, [id, print], [exps, [int, 1]]], [call, [id, print], [exps, [int, 2]]], [call, [id, print], [exps, [int, 3]]], [call, [id, print], [exps, [call, [id, print], [exps, [int, 3]]]]], [call, [id, print], [exps, [int, 1], [int, 3], [int, 5], [int, 7]]]]
