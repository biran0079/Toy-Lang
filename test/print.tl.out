1
2
3
3
none
1 3 5 7
